tests/scripts/thread-cert/node.py

```diff
@@ -507,15 +507,24 @@ def set_backbone_router(self, seqno=None, reg_delay=None, mlr_timeout=None):
         self.send_command(cmd)
         self._expect('Done')
 
-    def set_domain_prefix(self, prefix):
-        flags = 'prosD'
+    def set_domain_prefix(self, prefix, flags='prosD'):
         self.add_prefix(prefix, flags)
         self.register_netdata()
 
     def remove_domain_prefix(self, prefix):
         self.remove_prefix(prefix)
         self.register_netdata()
 
+    def set_dua_iid(self, iid):
+        cmd = 'dua iid {}'.format(iid)
```

Comment:
make sure iid is a hex string of  correct length?

Reply:
nice to have. but the cli could already notify the incorrectness if the `iid` is not valid?